src/control/mod.rs

```diff
@@ -950,6 +950,7 @@ bitflags::bitflags! {
     /// Limited to the values in [`ffi::drm_sys::DRM_MODE_PAGE_FLIP_FLAGS`],
     /// minus [`ffi::drm_sys::DRM_MODE_PAGE_FLIP_TARGET`] bits which are
     /// passed through [`PageFlipTarget`].
+    #[derive(Clone, Copy, Debug)]
```

Comment:
Should these also derive `Eq, PartialEq, Hash`? Besides that only the `Ord` traits are now missing that were present on `bitflags 1`, but they're not relevant for these types.

Reply:
Certainly wouldn't hurt.

Reply:
Done.